tests/baselines/CodeLlama_13b_Instruct_hf.json

```diff
@@ -0,0 +1,23 @@
+{
+    "gaudi2": {
+        "wikitext": {
+            "num_train_epochs": 2,
```

Comment:
maybe 1 epoch?

Reply:
Fixed num_epochs and batchsize as 48(max supported).